project3/index.html

```diff
@@ -0,0 +1,410 @@
+            style="stroke-width: 2px; fill: rgb(250, 250, 250);" />
+    </svg>
+    <h1>super hero resistry</h1>
+    <h2>With great power there must also come—great responsibility</h2>
+    <form action="">
+        <div class="box"></div>
+        <div class="content">
+            <!-- <h3 class="sr-only">Identity</h3> -->
+
+            <h3 class="identity">identity</h3>
+
+            <label for="real-name">Real Name</label>
+            <input id="real-name" type="text" required>
+
+            <label for="alias">Alias<input id="alias" type="text" required>
+            </label>
+
+
+            <label for="email">Email <input id="email" type="text">
+            </label>
+
+            <h3 class="personal">personal</h3>
+            <h3 class="characteristics">characteristics</h3>
+
+            <label for="age">Age<input id="age" type="number" required></label>
+
+            <fieldset>
+                <div class="categories">
+                    <legend>Category</legend>
+                    <p>Please select as many as apply to your origin and how you gained your powers.</p>
+
+                    <span class=" category">
+                        <input name="categroy" id="alien" type="checkbox">
+                        <label for="alen">Alien</label></span>
+
+                    <span class="category">
+                        <input name="category" id="cyborg" type="checkbox">
+                        <label for="cyborg">Cyborg</label></span>
+
+                    <span class="category">
+                        <input name="category" id="experiment" type="checkbox">
+                        <label for="experiment">Experiment</label></span>
+
+                    <span class="category">
+                        <input name="category" id="human" type="checkbox">
+                        <label for="human">Human</label></span>
+
+                    <span class="category">
+                        <input name="category" id="inhuman" type="checkbox">
+                        <label for="inhuman">Inhuman</label></span>
+
+                    <span class="category">
+                        <input name="category" id="mutant" type="checkbox">
+                        <label for="mutant">Mutant</label></span>
+                </div>
+            </fieldset>
+
+            <fieldset>
+                <div class="gender">
```

Comment:
Same note as above -- I expect this `div` is unnecessary since you have the `fieldset` as a wrapper element.